### ISSUE TITLE
FR: Change theme color in browser meta when dark mode is toggled

### ISSUE THREAD

**Issue by Shanzid01** [halfmoonui/halfmoon#6]
Have the meta theme color change when the user toggles between dark/light modes:
[https://webhint.io/docs/user-guide/hints/hint-meta-theme-color/](https://webhint.io/docs/user-guide/hints/hint-meta-theme-color/)

**Comment by halfmoonui** [comment]
> Have the meta theme color change when the user toggles between dark/light modes:
> https://webhint.io/docs/user-guide/hints/hint-meta-theme-color/

Correct me if I am wrong, but this would require JavaScript to swap out the `meta` tag with a new one when dark mode is toggled. Right?

**Comment by Shanzid01** [comment]
Yep! Precisely. So this would have to be something which is only there if someone includes the javascript file.
